services/grafana/client.go

```diff
@@ -300,7 +303,34 @@ func (c *Client) testDeleteUser(ctx context.Context, userID int, authHeaders htt
 	return c.do(ctx, "DELETE", "/api/admin/users/"+strconv.Itoa(userID), "", authHeaders, nil, nil)
 }
 
-func (c *Client) testCreateAPIKey(ctx context.Context, name string, role role, authHeaders http.Header) (int, string, error) {
+func (c *Client) CreateAdminAPIKey(ctx context.Context, name string) (int, string, error) {
+	headers, ok := metadata.FromIncomingContext(ctx)
+	if !ok {
+		return 0, "", fmt.Errorf("cannot get headers from metadata %v", headers)
+	}
+	fmt.Println(headers)
+	// get authorization from headers.
+	authorizationHeaders := headers.Get("Authorization")
+	fmt.Println(authorizationHeaders)
+	cookieHeaders := headers.Get("grpcgateway-cookie")
+	fmt.Println(cookieHeaders)
```

Comment:
i am guessing all these `fmt.Println` were for debugging and you forgot to remove them, if yes then please remove them otherwise maybe log it instead if we want to see this information

Reply:
yes, correct